src/models/db/account.js

```diff
@@ -37,6 +37,8 @@ class Account extends Model {
     if (data.id) {
       data.name = uri.parse(data.id, 'account').name.toLowerCase()
       delete data.id
+    } else {
+      data.name = data.name.toLowerCase()
```

Comment:
Yes

On Thu, Mar 17, 2016 at 3:33 PM, Chris Clark <notifications@github.com>
wrote:

> In src/models/db/account.js
> <https://github.com/interledger/five-bells-ledger/pull/203#discussion_r56588901>
> :
>
> > @@ -37,6 +37,8 @@ class Account extends Model {
> >      if (data.id) {
> >        data.name = uri.parse(data.id, 'account').name.toLowerCase()
> >        delete data.id
> > +    } else {
> > +      data.name = data.name.toLowerCase()
>
> Can you add a test case for PUTing an account that is missing both id and
> name? We need to make sure it isn't a 500 error.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/interledger/five-bells-ledger/pull/203/files/411f80904730435d9d048883d8bf9631cd72232f#r56588901>
>